WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserActivity.java

```diff
@@ -93,6 +82,15 @@ public void onCreate(Bundle savedInstanceState) {
 
         setContentView(R.layout.theme_browser_activity);
 
+        if (savedInstanceState == null) {
+            AnalyticsUtils.trackWithSiteDetails(Stat.THEMES_ACCESSED_THEMES_BROWSER, mSite);
```

Comment:
How do you feel about having this in `ActivityLauncher.viewCurrentBlogThemes` instead?

Reply:
I am merging this PR, if you agree with this, I'll take care of it in my branch.

Reply:
I think that makes more sense.